datumaro/components/extractor.py

```diff
@@ -126,7 +126,8 @@ def __iter__(self):
 @attrs
 class Label(Annotation):
     _type = AnnotationType.label
-    label = attrib(converter=int)
+    label = attrib(converter=int, default=0)
```

Comment:
Giving `label` a default value seems questionable - why would one need to create a `Label` without the label index?

Reply:
Right, copy-pasted.